.github/workflows/docs.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
-    - uses: josh146/sphinx-action@master
+    - uses: XanaduAI/sphinx-action@master

```

Comment:
Seems better to use the `XanaduAI` organisation here. What's the difference, and why was the one from @josh146 used before? Is it only an older version? 🤔

Reply:
`XanaduAI/sphinx-action@master` is a newly created fork from @josh146's action. The former uses `python3.8` whereas the latter `python3.7`.

Reply:
🤔

Reply:
@josh146, for context: `josh146/sphinx-action` uses `python3.7` but `readthedocs` requires `python3..8`. Since they both use `docs/requirements.txt`, the build of one of them will fail if we point to _the walrus_ wheel for python 3.7 or 3.8.

Using `python3.8` on `sphinx-action` ensures that the same python version is used across all the CI.

Reply:
ah got it! no worries @sduquemesa. Better to use the XanaduAI anyway :)